connexion/cli.py

```diff
@@ -71,7 +73,7 @@ def main():
               help='Enable validation of response values from operation handlers.',
               is_flag=True, default=False)
 @click.option('--strict-validation',
-              help='Enable strict validation of request payloads.',
+              help='Enable strict validation of endpoint parameters.',
```

Comment:
This configuration name is misleading. I think we should think about renaming it. I named in the config variable as `STRICT_PARAMETERS_VALIDATION` to be more specific. But I would mark this to be removed in a future 2.x version of Connexion.